src/test/extension.test.ts

```diff
@@ -1,15 +1,100 @@
+import { createStubExtensionContext, createStubSourcegraphAPI } from '@sourcegraph/extension-api-stubs'
 import expect from 'expect'
 import mock from 'mock-require'
-import { createMockSourcegraphAPI, projects } from './stubs'
 
-const sourcegraph = createMockSourcegraphAPI()
-// For modules importing Range/Location/URI/etc
+export const sourcegraph = createStubSourcegraphAPI()
+// For modules importing Range/Location/Position/URI/etc
 mock('sourcegraph', sourcegraph)
 
-import { decorateEditor, decorateLine, getDecorations } from '../extension'
+import { activate, decorateEditor, decorateLine, getDecorations } from '../extension'
+import { resolveSettings, SentryProject } from '../settings'
 
-describe('extension', () => {
-    it('works', () => void 0)
+describe('check for extension activation', () => {
+    const context = createStubExtensionContext()
+    it('activate extension', () => expect(activate(context)).toEqual(void 0))
```

Comment:
The test doesn't really test whether it activates the extension. The only thing you're testing here is that it does not throw (the return value is not useful to test).
Therefor this should be `it('does not throw`